cves/CVE-2015-6821.yml

```
@@ -131,7 +138,12 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    The file existed since at least 2012 when the lines that were removed were
+    added and a large portion of the file was re-written. Prior to that the 
+    problem coudln't have existed since the funcitonality was added in this 2012
+    commit. There were 240 files changed in this commit so its likely a few 
+    vulnerabilities were missed in it.
```

```suggestion
    added and a large portion of the file was re-written. Prior to that, the 
    problem couldn't have existed since the functionality was added in this 2012
    commit. There were 240 files changed in this commit so it's likely a few 
    vulnerabilities were missed in it.
```
some grammar and spelling fixes